install/kots/manifests/gitpod-installer-job.yaml

```diff
@@ -174,8 +184,7 @@ spec:
                 kubectl create secret docker-registry container-registry \
                   --namespace "{{repl Namespace }}" \
                   --from-file=.dockerconfigjson=/tmp/container-registry-secret \
-                  -o yaml --dry-run=client | \
-                  kubectl replace --namespace "{{repl Namespace }}" --force -f -
+                  -o yaml --dry-run=client >  "${GITPOD_OBJECTS}/templates/gitpod.yaml"
```

Comment:
Changed this to save the secret in the `gitpod.yaml`, instead of applying so that we can post process it if the user submits more docker configs through the new option

Reply:
Seems a reasonable approach. This means that we'll have a secret at the start of the YAML file, but that shouldn't be a problem as it's what we're doing anyway